.azure-pipelines/callGithubWorkflow.yaml

```diff
@@ -56,11 +56,11 @@ jobs:
         $branchName = "$(System.PullRequest.SourceBranch)"
         $targetBranch = "$(System.PullRequest.TargetBranch)"
         $pullRequestNumber = "$(System.PullRequest.PullRequestNumber)"
-        $isAutoGeneratedPR = [bool]($branchName -match "-automated-pr")
-        Write-Host "BranchName is $branchName, pullRequestNumber $pullRequestNumber, isAutoGeneratedPR $isAutoGeneratedPR"
+        $isAutoGeneratedOrDependabotPR = [bool]($branchName -match "dependabot/|-automated-pr")
```

Comment:
will the branch name for dependabot will start with this format "/dependabot/"?

Reply:
it starts with "dependabot/"